evaluations/2018_ACM_The-Maximum-Transmission-Switching-Flow-Problem/statistics/run_betweenness_DC_NESTA-Parallel.sh

```diff
@@ -0,0 +1,97 @@
+/Users/franziska-wegner/Documents/work/frameworks/egoa/.build/bin/dtpCentrality -i "/Users/franziska-wegner/Documents/work/frameworks/egoa/data/PowerGrids/nesta/dc_mtsf_cases/nesta_case2746wop_mp.m" -sf "/Users/franziska-wegner/Documents/work/paper/MTSF-ACM-e-Energy/02-ACM/data/results/DTP-Betweenness/SwitchingBetweenness_nesta_case2746wop_mp.csv" -a DTP -cr sm | /usr/bin/egrep -v "Academic license - for non-commercial use only"
+/Users/franziska-wegner/Documents/work/frameworks/egoa/.build/bin/dtpCentrality -i "/Users/franziska-wegner/Documents/work/frameworks/egoa/data/PowerGrids/nesta/dc_mtsf_cases/nesta_case2746wp_mp.m" -sf "/Users/franziska-wegner/Documents/work/paper/MTSF-ACM-e-Energy/02-ACM/data/results/DTP-Betweenness/SwitchingBetweenness_nesta_case2746wp_mp.csv" -a DTP -cr sm | /usr/bin/egrep -v "Academic license - for non-commercial use only"
+/Users/franziska-wegner/Documents/work/frameworks/egoa/.build/bin/dtpCentrality -i "/Users/franziska-wegner/Documents/work/frameworks/egoa/data/PowerGrids/nesta/dc_mtsf_cases/nesta_case3012wp_mp.m" -sf "/Users/franziska-wegner/Documents/work/paper/MTSF-ACM-e-Energy/02-ACM/data/results/DTP-Betweenness/SwitchingBetweenness_nesta_case3012wp_mp.csv" -a DTP -cr sm | /usr/bin/egrep -v "Academic license - for non-commercial use only"
+/Users/franziska-wegner/Documents/work/frameworks/egoa/.build/bin/dtpCentrality -i "/Users/franziska-wegner/Documents/work/frameworks/egoa/data/PowerGrids/nesta/dc_mtsf_cases/nesta_case3120sp_mp.m" -sf "/Users/franziska-wegner/Documents/work/paper/MTSF-ACM-e-Energy/02-ACM/data/results/DTP-Betweenness/SwitchingBetweenness_nesta_case3120sp_mp.csv" -a DTP -cr sm | /usr/bin/egrep -v "Academic license - for non-commercial use only"
+
+/Users/franziska-wegner/Documents/work/frameworks/egoa/.build/bin/dtpCentrality -i "/Users/franziska-wegner/Documents/work/frameworks/egoa/data/PowerGrids/nesta/dc_mtsf_cases/nesta_case29_edin.m" -sf "/Users/franziska-wegner/Documents/work/paper/MTSF-ACM-e-Energy/02-ACM/data/results/DTP-Betweenness/SwitchingBetweenness_nesta_case29_edin.csv" -a DTP -cr sm | /usr/bin/egrep -v "Academic license - for non-commercial use only"
+# # -> /Users/franziska-wegner/Documents/work/frameworks/egoa/.build/bin/dtpCentrality -i "/Users/franziska-wegner/Documents/work/frameworks/egoa/data/PowerGrids/nesta/dc_mtsf_cases/nesta_case3375wp_mp.m" -sf "/Users/franziska-wegner/Documents/work/paper/MTSF-ACM-e-Energy/02-ACM/data/results/DTP-Betweenness/SwitchingBetweenness_nesta_case3375wp_mp.csv" -a DTP -cr sm | /usr/bin/egrep -v "Academic license - for non-commercial use only"
```

Comment:
Simplify the above code similar to the previous scripts. Think of merging it with the single threaded script.